<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMessageBox</class>
 <widget class="QDialog" name="CMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnESC">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMoreText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>210</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>moretext</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../hiway.qrc">:/images/res/bg-inputbox.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../hiway.qrc">:/images/res/icon-ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>240</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <zorder>lblBackground</zorder>
  <zorder>lblMoreText</zorder>
  <zorder>lblIcon</zorder>
  <zorder>lblStatus</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnESC</zorder>
 </widget>
 <resources>
  <include location="../hiway.qrc"/>
 </resources>
 <connections/>
</ui>
